Add YouTube and Editorial team goals (#18861)
PostHog/posthog.com: contents/teams/editorial/objectives.mdx

**What we'll ship:** A data-heavy newsletter leveraging our first-party data that acts as a sort of minimum viable brand study. It should include some pretty graphs. 

**We'll know we're successful when:** We ship the newsletter, share the graphs on social, and have a good idea what it takes to produce one of these. 

> ## James' social leadership

Author: ```suggestion
## James social
```

**Owner:** <TeamMember name="Andy Vandervell" photo />

> **Motivation:** Make James famous for self-driving.

Author: ```suggestion
**Motivation:** Make PostHog famous for self-driving through James
```

> **What we'll ship:** Two posts a week from James sharing how we think about self-driving and our post-experts strategy.

Author: ```suggestion
**What we'll ship:** Two posts a week from James sharing how we think about self-driving, our strategy and ideas generally.
```

> **We'll know we're successful when:** Posts consistently perform well and we see cut-through — people talking about PostHog off the back of them.

Author: ```suggestion
**We'll know we're successful when:** Posts consistently perform well and we see cut-through — i.e. people talking about PostHog off the back of them.
```
